recipes/conversation_view/feed_pill/feed_item_pill.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div :class="[wrapperClass, expanded ? 'dt-feed-item--expanded-pill' : 'd-bar-pill']">
+    <div :class="['d-bgc-secondary', expanded ? 'dt-feed-item--expanded-pill' : 'd-bar-pill']">
+      <div class="d-p8">
+        <button
+          :aria-label="ariaLabel"
+          data-qa="dt-feed-item-pill"
+          :class="['d-baw0 d-bgc-moderate d-bar-pill d-w100p d-ta-left', buttonClass]"
+          @focusin="hover = true"
+          @focusout="hover = false"
+          @mouseenter="hover = true"
+          @mouseleave="hover = false"
+          @click="$emit('click')"
+        >
+          <dt-item-layout class="d-w100p d-p8">
+            {{ title }}
+            <template slot="left">
+              <dt-icon
+                v-if="toggleable && hover"
+                class="dt-feed-icon"
+                data-qa="dt-feed-item-hover-icon"
+                :name="expanded ? 'chevron-down' : 'chevron-right'"
+              />
+              <dt-icon
+                v-else
+                class="dt-feed-icon"
+                data-qa="dt-feed-item-icon"
+                :name="iconName"
+                size="300"
+              />
+            </template>
+            <template slot="subtitle">
+              <slot name="subtitle" />
+            </template>
+            <template slot="bottom">
+              <slot name="bottom" />
+            </template>
+            <template slot="right">
+              <slot name="right" />
+            </template>
+          </dt-item-layout>
+        </button>
+        <div
+          v-if="expanded"
+          class="d-jc-center d-d-flex"
+        >
+          <slot name="content" />
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { DtIcon, DtItemLayout } from '@/index';
+
+export default {
+  name: 'DtRecipeFeedItemPill',
+
+  components: { DtItemLayout, DtIcon },
+
+  mixins: [],
+
+  props: {
+    /**
+     * Accepts a DtIcon name to be shown in the left
+     */
+    iconName: {
+      type: String,
+      default: () => '',
+    },
+
+    /**
+     * Bolded primary text
+     */
+    title: {
+      type: String,
+      default: () => '',
+    },
+
+    /**
+     * Shows/hides the expanded portion
+     */
+    expanded: {
+      type: Boolean,
+      default: () => false,
+    },
```

Comment:
There is currently no "automatically expand on click" functionality right? It has to be controlled by the consumer capturing the click event and setting expanded? Should we have the default behaviour be to expand on click and then implement .sync functionality into expanded?

Reply:
or maybe even better, when toggleable is true it would automatically expand / collapse on click but when false it does nothing.